tools/ZAPD/README.md

```diff
@@ -16,6 +16,14 @@ In a Debian/Ubuntu based environment, those could be installed with the followin
 sudo apt install libpng-dev
 ```
 
+On a Mac, you will need to install libpng with Homebrew or MacPorts; we currently only support Homebrew. You can run
```

Comment:
```suggestion
On a Mac, you will need to install libpng with Homebrew. You can run
```

Since we don't support MacPorts I don't think mentioning it is necessary?

Reply:
This would have to be changed in the ZAPD repo.